src/app/task-list/task-list.component.ts

```diff
@@ -0,0 +1,38 @@
+import { Component, OnInit } from "@angular/core";
+
+import { TaskDTO } from "../shared/TaskDTO";
+
+@Component({
+	selector: "task-list",
+	templateUrl: "./task-list.component.html",
+	styleUrls: ["./task-list.component.css"]
+})
+export class TaskListComponent implements OnInit {
+
+	public taskList: TaskDTO[];
+
+	public constructor() {
+		this.taskList = [
+			{
+				id: 1,
+				title: "Title",
+				description: "Description",
+				isCompleted: false,
+				isDeleted: false,
+				userId: 1
+			},
+			{
+				id: 2,
+				title: "Title 2",
+				description: "Description 2",
+				isCompleted: false,
+				isDeleted: false,
+				userId: 1
+			}
+		];
+	}
+
+	public ngOnInit(): void {
+	}
```

Comment:
```suggestion
```

Reply:
i know that at the moment there's nothing inside ngOnInit, but we'll have to load the tasks here once we have the api